container/go/cmd/update_deps/BUILD

```diff
@@ -11,14 +11,26 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-package(default_visibility = ["//visibility:private"])
+# ###
+# Synthesize a .bzl file containing the digests for a given repository.
 
-py_binary(
+load("@bazel_gazelle//:def.bzl", "gazelle")
```

Comment:
buildifier is complaining this is unused

Reply:
Whoops, good catch. Thanks! Forgot to run buildifier before committing changes